src/components/ContactsList/ContactsList.jsx

```diff
@@ -2,21 +2,57 @@ import React from "react";
 import PropTypes from "prop-types";
 import { sortGivenNameFirst, getDisplayedName } from "./";
 import ContactRow from "./ContactRow";
+import { Button, IntentOpener } from "cozy-ui/react";
 
 const ContactHeaderRow = props => <div className="divider">{props.header}</div>;
 ContactHeaderRow.propTypes = {
   header: PropTypes.string.isRequired
 };
 
-const ContactsEmptyList = () => (
-  <div>
-    <p>No Contact</p>
-    <p>Try to create one with the big button on the top</p>
-    <p>
-      Or to import your contacts with this <a href="#">link</a>
-    </p>
-  </div>
-);
+class ContactsEmptyList extends React.Component {
+  state = {
+    hasConnector: false
+  };
+
+  componentWillMount() {
+    // cozy-client-js is needed for intents
+    const root = document.querySelector("[role=application]");
+    const data = root.dataset;
+    cozy.client.init({
+      cozyURL: `${window.location.protocol}//${data.cozyDomain}`,
+      token: data.cozyToken
+    });
+  }
+
+  afterConnection = () => {
+    this.setState({ hasConnector: true });
+  };
+
+  render() {
+    const { hasConnector } = this.state;
+    const { t } = this.context;
+
+    return (
+      <div className="list-empty">
+        <h1>{t("empty.title")}</h1>
+        <p>{t("empty.description")}</p>
+
+        {hasConnector ? (
+          <p>{t("empty.after")}</p>
+        ) : (
+          <IntentOpener
+            action="CREATE"
+            doctype="io.cozy.accounts"
+            options={{ slug: "google" }}
```

Comment:
SInce I don't have the konnector installed locally, I'm not 100% sure it works. But my intel says it will.

Reply:
I'll check it, stay tuned!